src/proxy/Records.js

```diff
@@ -5,6 +5,7 @@ Ext.define('Emergence.proxy.Records', {
     alias: 'proxy.records',
     requires: [
         'Emergence.util.API',
+        'Jarvus.proxy.API',
```

Comment:
@aharley40 the class does not need to require the class it's extending, remove this